scripts/get_native_libraries.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+import shutil
+import urllib.request
+import zipfile
+
+from os import path, remove
+
+
+def download_native_libraries(name: str, version: str, destination: str):
+    mac = f'{destination}/darwin-x86-64/libcosmian_{name}.dylib'
+    linux = f'{destination}/linux-x86-64/libcosmian_{name}.so'
+    windows = f'{destination}/win32-x86-64/cosmian_{name}.dll'
+
+    if (
+        not path.exists(mac)
+        or not path.exists(linux)
+        or not path.exists(windows)
+    ):
+        print(
+            f'Missing {name} native library. Copy {name} {version} to {destination}...'
+        )
+
+        url = f'https://package.cosmian.com/{name}/{version}/all.zip'
+        try:
+            r = urllib.request.urlopen(url)
+            if r.getcode() != 200:
+                download_native_libraries(name, 'last_build', destination)
```

Comment:
Same here, I think there should be no default, just fail and exit

Reply:
we need this temporary behaviour (where the Findex/CoverCrypt is not released yet). Once published, it will fetch the required version.

Reply:
can you put `last_build` below instead? Instead of the version and change to the version once Findex/CoverCrypt is released? (so we have the correct code and just have to change the version?)

Reply:
isn't it already below?

Reply:
On line 27 put a crash instead of `download_native_libraries(name, 'last_build', destination)` and temporary put `download_native_libraries(name, 'last_build', destination)` in `main`

Reply:
right